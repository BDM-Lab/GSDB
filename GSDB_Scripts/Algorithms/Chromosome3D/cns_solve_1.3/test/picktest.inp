remarks PICKTEST.INP
remarks TEST PICKING OF GEOMETRIC PROPERTIES
remarks by Axel Brunger
remarks ===============

topology @CNS_TOPPAR:protein.top end
parameter @CNS_TOPPAR:protein.param end


segment
  name=tryp
  chain
     @CNS_TOPPAR:protein.link
     sequence
        TRP
     end
  end
end

coor
ATOM      1  HT1 TRP     1       0.101   0.574  -1.385  1.00  0.00      TRYP
ATOM      2  HT2 TRP     1       1.711   0.436  -1.904  1.00  0.00      TRYP
ATOM      3  N   TRP     1       1.067   0.302  -1.101  1.00  0.00      TRYP
ATOM      4  HT3 TRP     1       1.080  -0.689  -0.791  1.00  0.00      TRYP
ATOM      5  CA  TRP     1       1.472   1.192   0.027  1.00  0.00      TRYP
ATOM      6  CB  TRP     1       2.696   2.026  -0.348  1.00  0.00      TRYP
ATOM      7  CG  TRP     1       3.922   1.169  -0.679  1.00  0.00      TRYP
ATOM      8  CD2 TRP     1       4.295   0.679  -1.890  1.00  0.00      TRYP
ATOM      9  CE2 TRP     1       5.424  -0.010  -1.601  1.00  0.00      TRYP
ATOM     10  CE3 TRP     1       3.769   0.774  -3.155  1.00  0.00      TRYP
ATOM     11  CD1 TRP     1       4.813   0.775   0.290  1.00  0.00      TRYP
ATOM     12  NE1 TRP     1       5.727   0.046  -0.320  1.00  0.00      TRYP
ATOM     13  HE1 TRP     1       6.094  -0.743   0.127  1.00  0.00      TRYP
ATOM     14  CZ2 TRP     1       6.138  -0.677  -2.563  1.00  0.00      TRYP
ATOM     15  CZ3 TRP     1       4.472   0.108  -4.148  1.00  0.00      TRYP
ATOM     16  CH2 TRP     1       5.636  -0.605  -3.856  1.00  0.00      TRYP
ATOM     17  C   TRP     1       0.280   2.113   0.309  1.00  0.00      TRYP
ATOM     18  O   TRP     1      -0.402   2.481  -0.636  1.00  0.00      TRYP
ATOM     19  OXT TRP     1      -0.106   2.192   1.459  1.00  0.00      TRYP
END

set wrnlev=4 end

coor copy end

parameter 
  nbonds
     atom cdie shift eps=1.0
     cutnb=30.0 ctonnb=99.0 ctofnb=99.0
     nbxmod=5 vswitch 
  end
end


{* just print bonds, angles,... *}
{* ============================ *}
print bonds
display  $rms  $violations
print angles
display  $rms  $violations
print dihedrals
display  $rms  $violations
print impropers
display  $rms  $violations

print threshold=0.1 bonds
display  $rms  $violations
print threshold=10.0 angles
display  $rms  $violations
print threshold=40.0 dihedrals
display  $rms  $violations
print threshold=20.0 impropers
display  $rms  $violations


{* test individual picking *}
{* ======================= *}
pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) geometry
display  $result
pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) energy
display  $result
pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) equilibrium
display  $result
pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) delta
display  $result
pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) constant
display  $result
pick bond ( atom tryp 1 c* ) ( atom tryp 1 n* ) geometry
display  $result

pick angle ( atom tryp 1 ca ) ( atom tryp 1 cb ) ( atom tryp 1 cg ) geometry
display  $result

pick dihedral 
 (atom tryp 1 n) (atom tryp 1 ca) (atom tryp 1 cb) (atom tryp 1 cg) geometry 
display  $result
pick dihedral 
 (atom tryp 1 n) (atom tryp 1 ca) (atom tryp 1 cb) (atom tryp 1 cg) energy
display  $result

pick improper
 (atom tryp 1 ca) (atom tryp 1 n) (atom tryp 1 c) (atom tryp 1 cb) geometry
display  $result
pick improper
 (atom tryp 1 ca) (atom tryp 1 n) (atom tryp 1 c) (atom tryp 1 cb) energy
display  $result

pick ring
  (name ca)(name cb)(name cg)(name cd2)(name c) pucker
display  $n_amplitude $n_phase 
display  $amplitude1 $phase1

pick ring
  (name ca)(name cb)(name cg)(name cd2)(name c) (name n ) pucker
display  $n_amplitude $n_phase 
display  $amplitude1 $phase1
display  $amplitude2 


stop


