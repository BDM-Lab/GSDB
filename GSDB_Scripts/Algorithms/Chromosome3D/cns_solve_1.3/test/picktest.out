          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:05 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks PICKTEST.INP 
 CNSsolve>remarks TEST PICKING OF GEOMETRIC PROPERTIES 
 CNSsolve>remarks by Axel Brunger 
 CNSsolve>remarks =============== 
 CNSsolve> 
 CNSsolve>topology @CNS_TOPPAR:protein.top end 
 ASSFIL: file protein.top opened.
 RTFRDR>remarks file toppar/protein.top 
 RTFRDR>remarks   protein topology with polar hydrogens 
 RTFRDR> 
 RTFRDR>! 
 RTFRDR>! Please cite the following reference when using these parameters: 
 RTFRDR>! Engh, R.A. and  Huber, R. (1991). Accurate Bond and 
 RTFRDR>!  Angle Parameters for X-ray Protein-Structure Refinement, 
 RTFRDR>!  Acta Cryst. A47, 392-400. 
 RTFRDR>! 
 RTFRDR>! 
 RTFRDR> 
 RTFRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 RTFRDR>set message=off echo=off end 
 RTFRDR> 
 RTFRDR> 
 RTFRDR> end 
 CNSsolve>parameter @CNS_TOPPAR:protein.param end 
 ASSFIL: file protein.param opened.
 PARRDR>remarks file toppar/protein.param 
 PARRDR>remarks   protein parameters with Lennard-Jones van der Waals parameters 
 PARRDR>remarks   for crystallographic structure determination 
 PARRDR> 
 PARRDR>! 
 PARRDR>! Please cite the following reference when using these parameters: 
 PARRDR>! Engh, R.A. and  Huber, R. (1991). Accurate Bond and 
 PARRDR>!  Angle Parameters for X-ray Protein-Structure Refinement, 
 PARRDR>!  Acta Cryst. A47, 392-400. 
 PARRDR>! 
 PARRDR>! 
 PARRDR> 
 PARRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 PARRDR>set message=off echo=off end 
 PARRDR> 
 PARRDR> end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>  name=tryp 
 SEGMENT>  chain 
 CHAIN>     @CNS_TOPPAR:protein.link 
 ASSFIL: file protein.link opened.
 CHAIN>remarks file toppar/protein.link 
 CHAIN>remarks 
 CHAIN>remarks this is a macro to define standard protein peptide bonds 
 CHAIN>remarks and termini to generate a protein sequence. 
 CHAIN> 
 CHAIN>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 CHAIN>set message=off echo=off end 
 CHAIN>     sequence 
 SEQUENCE>        TRP 
 SEQUence-element (terminate with END) =     end 
 SEGMNT:     1 residues were inserted into segment "TRYP"
 CHAIN>  end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         19(MAXA=     1000000)  NBOND=         20(MAXB=     1000000)
 -> NTHETA=        30(MAXT=     2000000)  NGRP=           3(MAXGRP=   1000000)
 -> NPHI=          10(MAXP=     2000000)  NIMPHI=        15(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>coor 
 COOR>ATOM      1  HT1 TRP     1       0.101   0.574  -1.385  1.00  0.00      TRYP 
 COOR>ATOM      2  HT2 TRP     1       1.711   0.436  -1.904  1.00  0.00      TRYP 
 CNSsolve> 
 CNSsolve>set wrnlev=4 end 
 CNSsolve> 
 CNSsolve>coor copy end 
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>  nbonds 
 NBDSET>     atom cdie shift eps=1.0 
 NBDSET>     cutnb=30.0 ctonnb=99.0 ctofnb=99.0 
 NBDSET>     nbxmod=5 vswitch 
 NBDSET>  end 
 %NBDSET-ERR: inconsistent CTONNB, CTOFNB given. Reset.
 %NBDSET-ERR: inconsistent CUTNB, TOLErance, CTOFNB given.  Reset.
 PARRDR>end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>{* just print bonds, angles,... *} 
 CNSsolve>{* ============================ *} 
 CNSsolve>print bonds 
 (atom-i        |atom-j        )    dist.   equil.   delta    energy   const. 

 (TRYP 1    N   |TRYP 1    CA  )    1.493    1.491    0.002    0.004 1342.404
 (TRYP 1    CA  |TRYP 1    C   )    1.533    1.525    0.008    0.076 1342.404
 (TRYP 1    C   |TRYP 1    O   )    1.222    1.249   -0.027    1.185 1639.889
 (TRYP 1    CA  |TRYP 1    CB  )    1.528    1.530   -0.002    0.007 1480.000
 (TRYP 1    CB  |TRYP 1    CG  )    1.532    1.498    0.034    0.713  616.025
 (TRYP 1    CG  |TRYP 1    CD1 )    1.374    1.365    0.009    0.078  947.200
 (TRYP 1    CG  |TRYP 1    CD2 )    1.359    1.433   -0.074   10.118 1827.161
 (TRYP 1    CD1 |TRYP 1    NE1 )    1.319    1.374   -0.055    4.107 1342.404
 (TRYP 1    CD2 |TRYP 1    CE2 )    1.354    1.409   -0.055    6.233 2048.443
 (TRYP 1    NE1 |TRYP 1    HE1 )    0.978    0.980   -0.002    0.001  405.000
 (TRYP 1    NE1 |TRYP 1    CE2 )    1.318    1.370   -0.052   13.466 4892.562
 (TRYP 1    CD2 |TRYP 1    CE3 )    1.373    1.398   -0.025    1.412 2312.500
 (TRYP 1    CE2 |TRYP 1    CZ2 )    1.371    1.394   -0.023    0.699 1342.404
 (TRYP 1    CE3 |TRYP 1    CZ3 )    1.387    1.382    0.005    0.017  657.778
 (TRYP 1    CZ2 |TRYP 1    CH2 )    1.389    1.368    0.021    0.716 1639.889
 (TRYP 1    CZ3 |TRYP 1    CH2 )    1.396    1.400   -0.004    0.016  947.200
 (TRYP 1    HT1 |TRYP 1    N   )    1.043    1.040    0.003    0.004  405.000
 (TRYP 1    HT2 |TRYP 1    N   )    1.038    1.040   -0.002    0.002  405.000
 (TRYP 1    HT3 |TRYP 1    N   )    1.038    1.040   -0.002    0.001  405.000
 (TRYP 1    C   |TRYP 1    OXT )    1.216    1.249   -0.033    1.827 1639.889
 RMS deviation=   0.031
 CNSsolve>display  $rms  $violations 
 3.095265E-02  20
 CNSsolve>print angles 
 (atom-i        |atom-j        |atom-k        )  angle    equil.     delta    energy  const. 

 (TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   )  106.637  111.200   -4.563    1.572  247.886
 (TRYP 1    N   |TRYP 1    CA  |TRYP 1    CB  )  110.936  110.500    0.436    0.039  672.465
 (TRYP 1    CA  |TRYP 1    C   |TRYP 1    O   )  118.211  117.000    1.211    0.139  310.948
 (TRYP 1    C   |TRYP 1    CA  |TRYP 1    CB  )  109.891  110.100   -0.209    0.007  538.345
 (TRYP 1    CA  |TRYP 1    CB  |TRYP 1    CG  )  112.878  113.600   -0.722    0.085  538.345
 (TRYP 1    CB  |TRYP 1    CG  |TRYP 1    CD1 )  121.800  126.900   -5.100    6.845  863.744
 (TRYP 1    CB  |TRYP 1    CG  |TRYP 1    CD2 )  127.883  126.800    1.083    0.354  991.543
 (TRYP 1    CD1 |TRYP 1    CG  |TRYP 1    CD2 )  110.313  106.300    4.013    3.724  759.150
 (TRYP 1    CG  |TRYP 1    CD1 |TRYP 1    NE1 )  106.364  110.200   -3.836    5.154 1149.956
 (TRYP 1    CG  |TRYP 1    CD2 |TRYP 1    CE2 )  102.840  107.200   -4.360    7.815 1349.600
 (TRYP 1    CG  |TRYP 1    CD2 |TRYP 1    CE3 )  133.707  133.900   -0.193    0.022 1943.424
 (TRYP 1    CD1 |TRYP 1    NE1 |TRYP 1    HE1 )  119.638  120.000   -0.362    0.001   35.000
 (TRYP 1    CD1 |TRYP 1    NE1 |TRYP 1    CE2 )  108.291  108.900   -0.609    0.068  599.823
 (TRYP 1    CD2 |TRYP 1    CE2 |TRYP 1    NE1 )  112.191  107.400    4.791    8.041 1149.956
 (TRYP 1    CE2 |TRYP 1    CD2 |TRYP 1    CE3 )  123.453  118.800    4.653   12.816 1943.424
 (TRYP 1    CD2 |TRYP 1    CE2 |TRYP 1    CZ2 )  122.143  122.400   -0.257    0.039 1943.424
 (TRYP 1    HE1 |TRYP 1    NE1 |TRYP 1    CE2 )  119.752  120.000   -0.248    0.001   30.000
 (TRYP 1    NE1 |TRYP 1    CE2 |TRYP 1    CZ2 )  125.666  130.100   -4.434    5.174  863.744
 (TRYP 1    CD2 |TRYP 1    CE3 |TRYP 1    CZ3 )  115.602  118.600   -2.998    3.148 1149.956
 (TRYP 1    CE2 |TRYP 1    CZ2 |TRYP 1    CH2 )  116.086  117.500   -1.414    0.700 1149.956
 (TRYP 1    CE3 |TRYP 1    CZ3 |TRYP 1    CH2 )  121.208  121.100    0.108    0.004 1149.956
 (TRYP 1    CZ2 |TRYP 1    CH2 |TRYP 1    CZ3 )  121.508  121.500    0.008    0.000 1149.956
 (TRYP 1    HT1 |TRYP 1    N   |TRYP 1    HT2 )  109.288  109.500   -0.212    0.001   40.000
 (TRYP 1    HT2 |TRYP 1    N   |TRYP 1    HT3 )  110.265  109.500    0.765    0.007   40.000
 (TRYP 1    HT2 |TRYP 1    N   |TRYP 1    CA  )  109.831  109.500    0.331    0.001   35.000
 (TRYP 1    HT1 |TRYP 1    N   |TRYP 1    HT3 )  109.984  109.500    0.484    0.003   40.000
 (TRYP 1    HT1 |TRYP 1    N   |TRYP 1    CA  )  107.551  109.500   -1.949    0.041   35.000
 (TRYP 1    HT3 |TRYP 1    N   |TRYP 1    CA  )  109.876  109.500    0.376    0.002   35.000
 (TRYP 1    CA  |TRYP 1    C   |TRYP 1    OXT )  117.394  117.000    0.394    0.015  310.948
 (TRYP 1    O   |TRYP 1    C   |TRYP 1    OXT )  122.326  122.900   -0.574    0.034  337.400
 RMS deviation=   2.459
 CNSsolve>display  $rms  $violations 
 2.45889  30
 CNSsolve>print dihedrals 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TRYP 1    N   |TRYP 1    CA  |TRYP 1    CB  |TRYP 1    CG  )  -60.895  -60.000    0.895    0.005    4.800   3
 (TRYP 1    CA  |TRYP 1    CB  |TRYP 1    CG  |TRYP 1    CD1 )  -88.903  -60.000   28.903    0.000    0.000   3
 (TRYP 1    CD1 |TRYP 1    CE2 |TRYP 1    CD2 |TRYP 1    CZ3 )  179.987  180.000    0.013    0.000  300.000   2
 (TRYP 1    CD1 |TRYP 1    CD2 |TRYP 1    CE2 |TRYP 1    CH2 )  179.984  180.000    0.016    0.000  300.000   2
 (TRYP 1    CZ2 |TRYP 1    CE2 |TRYP 1    CD2 |TRYP 1    CG  )  179.944  180.000    0.056    0.001  300.000   2
 (TRYP 1    CE3 |TRYP 1    CD2 |TRYP 1    CE2 |TRYP 1    NE1 )  179.989  180.000    0.011    0.000  300.000   2
 (TRYP 1    HT2 |TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   )  119.966   60.000  -59.966    3.600    1.800   3
 (TRYP 1    HT1 |TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   )    1.128   60.000   58.872    3.597    1.800   3
 (TRYP 1    HT3 |TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   ) -118.575  -60.000   58.575    3.595    1.800   3
 (TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   |TRYP 1    OXT )  129.029  180.000   50.971    0.000    0.000   3
 RMS deviation=  37.319
 CNSsolve>display  $rms  $violations 
 37.319  10
 CNSsolve>print impropers 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TRYP 1    CA  |TRYP 1    N   |TRYP 1    C   |TRYP 1    CB  )   34.635   35.264    0.630    0.036  300.000   0
 (TRYP 1    CG  |TRYP 1    CD1 |TRYP 1    CD2 |TRYP 1    CB  )   -0.404    0.000    0.404    0.037  750.000   0
 (TRYP 1    NE1 |TRYP 1    CD1 |TRYP 1    CE2 |TRYP 1    HE1 )   18.074    0.000  -18.074   13.433  135.000   0
 (TRYP 1    CD2 |TRYP 1    CE2 |TRYP 1    CZ2 |TRYP 1    CH2 )    0.034    0.000   -0.034    0.000  750.000   0
 (TRYP 1    CE2 |TRYP 1    CZ2 |TRYP 1    CH2 |TRYP 1    CZ3 )    0.024    0.000   -0.024    0.000  750.000   0
 (TRYP 1    CZ2 |TRYP 1    CH2 |TRYP 1    CZ3 |TRYP 1    CE3 )   -0.023    0.000    0.023    0.000  750.000   0
 (TRYP 1    CH2 |TRYP 1    CZ3 |TRYP 1    CE3 |TRYP 1    CD2 )   -0.033    0.000    0.033    0.000  750.000   0
 (TRYP 1    CZ3 |TRYP 1    CE3 |TRYP 1    CD2 |TRYP 1    CE2 )    0.093    0.000   -0.093    0.002  750.000   0
 (TRYP 1    CE3 |TRYP 1    CD2 |TRYP 1    CE2 |TRYP 1    CZ2 )   -0.097    0.000    0.097    0.002  750.000   0
 (TRYP 1    CG  |TRYP 1    CD1 |TRYP 1    NE1 |TRYP 1    CE2 )    0.102    0.000   -0.102    0.000   75.000   0
 (TRYP 1    CD1 |TRYP 1    NE1 |TRYP 1    CE2 |TRYP 1    CD2 )   -0.085    0.000    0.085    0.002  750.000   0
 (TRYP 1    NE1 |TRYP 1    CE2 |TRYP 1    CD2 |TRYP 1    CG  )    0.029    0.000   -0.029    0.000  300.000   0
 (TRYP 1    CE2 |TRYP 1    CD2 |TRYP 1    CG  |TRYP 1    CD1 )    0.035    0.000   -0.035    0.000  750.000   0
 (TRYP 1    CD2 |TRYP 1    CG  |TRYP 1    CD1 |TRYP 1    NE1 )   -0.086    0.000    0.086    0.001  300.000   0
 (TRYP 1    C   |TRYP 1    CA  |TRYP 1    OXT |TRYP 1    O   )    8.913    0.000   -8.913    7.260  300.000   0
 RMS deviation=   5.207
 CNSsolve>display  $rms  $violations 
 5.20712  15
 CNSsolve> 
 CNSsolve>print threshold=0.1 bonds 
 (atom-i        |atom-j        )    dist.   equil.   delta    energy   const. 

 Number of violations greater    0.100:     0
 RMS deviation=   0.031
 CNSsolve>display  $rms  $violations 
 3.095265E-02  0
 CNSsolve>print threshold=10.0 angles 
 (atom-i        |atom-j        |atom-k        )  angle    equil.     delta    energy  const. 

 Number of violations greater   10.000:     0
 RMS deviation=   2.459
 CNSsolve>display  $rms  $violations 
 2.45889  0
 CNSsolve>print threshold=40.0 dihedrals 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 (TRYP 1    HT2 |TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   )  119.966   60.000  -59.966    3.600    1.800   3
 (TRYP 1    HT1 |TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   )    1.128   60.000   58.872    3.597    1.800   3
 (TRYP 1    HT3 |TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   ) -118.575  -60.000   58.575    3.595    1.800   3
 (TRYP 1    N   |TRYP 1    CA  |TRYP 1    C   |TRYP 1    OXT )  129.029  180.000   50.971    0.000    0.000   3
 Number of violations greater   40.000:     4
 RMS deviation=  37.319
 CNSsolve>display  $rms  $violations 
 37.319  4
 CNSsolve>print threshold=20.0 impropers 
 (atom-i        |atom-j        |atom-k        |atom-L        )    angle    equil.   delta    energy   const.   period

 Number of violations greater   20.000:     0
 RMS deviation=   5.207
 CNSsolve>display  $rms  $violations 
 5.20712  0
 CNSsolve> 
 CNSsolve> 
 CNSsolve>{* test individual picking *} 
 CNSsolve>{* ======================= *} 
 CNSsolve>pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) geometry 
 CNSsolve>display  $result 
 1.52786
 CNSsolve>pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) energy 
 CNSsolve>display  $result 
 6.776107E-03
 CNSsolve>pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) equilibrium 
 CNSsolve>display  $result 
 1.53
 CNSsolve>pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) delta 
 CNSsolve>display  $result 
 -2.139732E-03
 CNSsolve>pick bond ( atom tryp 1 ca ) ( atom tryp 1 cb ) constant 
 CNSsolve>display  $result 
 1480
 CNSsolve>pick bond ( atom tryp 1 c* ) ( atom tryp 1 n* ) geometry 
 CNSsolve>display  $result 
 1.15373
 CNSsolve> 
 CNSsolve>pick angle ( atom tryp 1 ca ) ( atom tryp 1 cb ) ( atom tryp 1 cg ) geometry 
 CNSsolve>display  $result 
 112.878
 CNSsolve> 
 CNSsolve>pick dihedral 
 SELRPN> (atom tryp 1 n) (atom tryp 1 ca) (atom tryp 1 cb) (atom tryp 1 cg) geometry 
 CNSsolve>display  $result 
 -60.8946
 CNSsolve>pick dihedral 
 SELRPN> (atom tryp 1 n) (atom tryp 1 ca) (atom tryp 1 cb) (atom tryp 1 cg) energy 
 CNSsolve>display  $result 
 5.264595E-03
 CNSsolve> 
 CNSsolve>pick improper 
 SELRPN> (atom tryp 1 ca) (atom tryp 1 n) (atom tryp 1 c) (atom tryp 1 cb) geometry 
 CNSsolve>display  $result 
 34.6347
 CNSsolve>pick improper 
 SELRPN> (atom tryp 1 ca) (atom tryp 1 n) (atom tryp 1 c) (atom tryp 1 cb) energy 
 CNSsolve>display  $result 
 3.623578E-02
 CNSsolve> 
 CNSsolve>pick ring 
 SELRPN>  (name ca)(name cb)(name cg)(name cd2)(name c) pucker 
 PUCKer=       0.691400      13.429440
 CNSsolve>display  $n_amplitude $n_phase 
 1 1
 CNSsolve>display  $amplitude1 $phase1 
 0.6914 13.4294
 CNSsolve> 
 CNSsolve>pick ring 
 SELRPN>  (name ca)(name cb)(name cg)(name cd2)(name c) (name n ) pucker 
 PUCKer=       1.160123      -0.258630      66.920741
 CNSsolve>display  $n_amplitude $n_phase 
 2 1
 CNSsolve>display  $amplitude1 $phase1 
 1.16012 66.9207
 CNSsolve>display  $amplitude2 
 -0.25863
 CNSsolve> 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8252216 bytes
           Maximum dynamic memory overhead:           288 bytes
           Program started at: 16:08:05 on 05-Jul-2018
           Program stopped at: 16:08:05 on 05-Jul-2018
           CPU time used:       0.0206 seconds
          ============================================================
